Makefile

```diff
@@ -1,5 +1,5 @@
 # The idiomatic way to disable test caching explicitly is to use -count=1
-GOTEST=go test -count=1
+GOTEST=go test -count=1 -race
```

Comment:
enable race conditions checks in the nitrite test.  Now that we have removed ionrocks, I do not think there are any other places using goroutimes, but it can be tricky to be sure.  Seems worth enabling, but happy to take it out if you think it is not needed.

Reply:
To clarify, the ionrocks lib introduces a race condition? Bummer. Your solution looks just as easy to use though!

Reply:
Nevermind, you described the race condition in your delphi PR

Reply:
Thanks!!